bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/XToPsiNSG/WH_HToJPsiG_JPsiToMuMu/WH_HToJPsiG_JPsiToMuMu_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+set default_unset_couplings 99
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model jpsi_sm
+add model sm
+define p   = g u c d s u~ c~ d~ s~
+define all = g u c d s t b u~ c~ d~ s~ t~ b~ ve vm vt e- mu- ta- ve~ vm~ vt~ e+ mu+ ta+
+generate    p p > w+ > w+ h, (h > jpsi a, jpsi > mu+ mu-), (w+ > all all)
+add process p p > w- > w- h, (h > jpsi a, jpsi > mu+ mu-), (w- > all all)
```

Comment:
The following syntax might be better (as WH is usually produced)
```
generate    p p > w+ h, (h > jpsi a, jpsi > mu+ mu-), (w+ > all all)
add process p p > w- h, (h > jpsi a, jpsi > mu+ mu-), (w- > all all)
```

Reply:
I will adapt it as you suggest. I agree it is cleaner